dashboard/test/ui/features/pd/workshopCertificates.feature

```diff
@@ -3,6 +3,7 @@
 
 Feature: Basic appearance for Workshop Certificates
 
+@no_circle
```

Comment:
This scenario failed in an earlier circle run: https://circleci.com/gh/code-dot-org/code-dot-org/35586